app/src/androidTest/java/com/pindroid/test/util/DateUtilsTest.java

```diff
@@ -73,10 +73,4 @@ public void testDateTimeParsing(){
 		} catch (Exception e) {
 		}
 	}
-	
-	public void testToString(){		
-		Date d = new Date(1133155569000l);
-
-		assertEquals("2005-11-28T05:26:09Z", DateParser.toString(d));

```

Comment:
the `DateParser.toString` method was removed few month ago, I guess you just forgot to remove the test